zebedee/client/client.go

```diff
@@ -239,3 +236,13 @@ func (c *ZebedeeClient) GetPageTitle(uri string) (data.PageTitle, error) {
 
 	return pt, nil
 }
+
+func (c *ZebedeeClient) createRequestURL(ctx context.Context, path string) string {
+	var url string
+	if ctx.Value("Collection-Id") != nil {
```

Comment:
Same here - use the existing context key and check the cast & log if its unsuccessful